Dockerfile

```diff
@@ -1,23 +1,47 @@
-FROM ubuntu:20.04
+# --------------------------------------------------
+# Base container
+# --------------------------------------------------
+FROM docker.io/ubuntu:focal AS base
 
 RUN set -e
 
-# Setup time-zone so that the build does not hang
-# on configuring the tzdata package.
-# I work in Brno, that is basically Vienna-North :)
-# (definitely its closer than Prague)
-ENV TZ=Europe/Vienna
-RUN ln -snf "/usr/share/zoneinfo/$TZ" /etc/localtime
-RUN echo "$TZ" > /etc/timezone
-
 # Install packages
+ENV DEBIAN_FRONTEND=noninteractive
 RUN apt-get update
-RUN apt-get install -y git cmake make llvm zlib1g-dev clang g++ python3
+RUN apt-get install -yq --no-install-recommends clang llvm
+
+# --------------------------------------------------
+# Build container
+# --------------------------------------------------
+FROM base as build
+
+# Install build dependencies
+RUN apt-get install -yq --no-install-recommends ca-certificates cmake git \
+                                                ninja-build llvm-dev python3
 
-WORKDIR /opt
+# Clone
 RUN git clone https://github.com/mchalupa/dg
-WORKDIR /opt/dg
-RUN cmake . -DCMAKE_INSTALL_PREFIX=/usr
-RUN make -j2
-RUN make check -j2
-RUN make install
+
+# Build
+WORKDIR /dg
```

Comment:
I think this still does not comply with the Filesystem Hierarchy Standard. Could we build DG in `/tmp` or `/var/tmp` (if we want to preserve source code)? Or somewhere like `/opt/dg-src` or `/opt/dg-build`?

Reply:
To be hones, this does not really matter as the build happens in a temporary container.

Reply:
> To be honest, this does not really matter as the build happens in a temporary container.

Sorry, I missed that build and release containers are separated. Let's merge it then.